charts/woodpecker/charts/server/values.yaml

```diff
@@ -39,13 +39,15 @@ env:
 extraSecretNamesForEnvFrom:
   # - woodpecker-github-client
   # - woodpecker-github-secret
-  - woodpecker-secret
+  - woodpecker-default-agent-secret # created by `createAgentSecret: true`
```

Comment:
Does it make sense to rely on `extraSecretNamesForEnvFrom` here? If `createAgentSecret: true` is set, why not create and use the secret anyway, regardless if it is present in this list or not?

Reply:
Fair point! We can map it directly into the STS without relying on this variable.